test/automation/src/electron.ts

```diff
@@ -47,14 +52,18 @@ export async function resolveElectronConfiguration(options: LaunchOptions): Prom
 		args[0] = `--${workspacePath.endsWith('.code-workspace') ? 'file' : 'folder'}-uri=vscode-remote://test+test/${URI.file(workspacePath).path}`;
 
 		if (codePath) {
+			if (!extensionsPath) {
+				throw new Error('Extensions path is required when running against a build at the moment.');
```

Comment:
It's unlikely that this code path will ever get hit, so I deferred getting it working for now.